### ISSUE TITLE
Cannot access Sitemap config in Backend on October 2

### ISSUE THREAD

**Issue by tobias-kuendig** [rainlab/sitemap-plugin#87]
I am currently unable to visit `/backend/rainlab/sitemap/definitions/update/1` on a October 2.1/2.2 installation:

> "Attempt to read property "theme" on null" on line 73 of plugins/rainlab/sitemap/controllers/Definitions.php

The `$model` is empty:
https://github.com/rainlab/sitemap-plugin/blob/master/controllers/Definitions.php#L72

**Comment by daftspunk** [comment]
Added some exception handling in 4bc4d8add0ebe9ef99566e76e96b566afff480f1

However, this will just make a nicer error message. This scenario is never meant to happen since a model is created and should always exist

```
$model = Definition::firstOrCreate(['theme' => $theme->getDirName()]);
```

Any more information you can provide would be useful.

**Comment by tobias-kuendig** [comment]
Hm, that's strange. I have this error on two different installations (2.2). The model is created correctly, I can see it in the database.

Adding this workaround after [modules/backend/widgets/form/HasFormWidgets.php:56](https://github.com/octobercms/october-private/blob/7c28fdfba81730596e2aee3edd44b7f8b7416301/modules/backend/widgets/form/HasFormWidgets.php#L54) fixes the issue:

```php
        if ($widgetClass === 'rainlab\sitemap\formwidgets\sitemapitems') {
            $widgetClass = 'RainLab\Sitemap\FormWidgets\SitemapItems';
        }
```

The `class_exists` call that follows seems to be case-sensitive (maybe this is a Linux/Windows problem?) and the widget class name is provided in lowercase and therefore not found. This throws the `SystemException`, which somewhere gets swallowed and `formGetModel()` then returns `null`. The [`initForm method returns here because of the exception`](https://github.com/octobercms/october-private/blob/7c28fdfba81730596e2aee3edd44b7f8b7416301/modules/backend/behaviors/FormController.php#L177) so the model property is never set.

Here's the [`$widgetConfig`](https://github.com/octobercms/october-private/blob/7c28fdfba81730596e2aee3edd44b7f8b7416301/modules/backend/widgets/form/HasFormWidgets.php#L45) from my machine. It looks like the `type` property is right, but `widget` is all lower-case:

![grafik](https://user-images.githubusercontent.com/8600029/156517197-c39d0403-146e-4743-853f-1561ff0b6461.png)

**Comment by daftspunk** [comment]
Ah, yes, there was some mysterious behavior like this because the types were getting lowercased. See if this fixes it https://github.com/octobercms/library/commit/b594c7001db2c63ac7dcf1fd2e75c8fac82997c9 (released in v2.2.5)

**Comment by tobias-kuendig** [comment]
Looking good, thank you!
